oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/ReactiveJwtDecoders.java

```diff
@@ -49,11 +49,12 @@ private ReactiveJwtDecoders() {
 	 * @return a {@link ReactiveJwtDecoder} that was initialized by the OpenID Provider
 	 * Configuration.
 	 */
-	public static ReactiveJwtDecoder fromOidcIssuerLocation(String oidcIssuerLocation) {
+	@SuppressWarnings("unchecked")
```

Comment:
Will you please update the copyright year to `2024` at the top of the file?

Reply:
I changed commit, thanks for your review